### PR TITLE
sops: add page

### PR DESCRIPTION
**Pull request by mkatychev**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/sops.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+> SOPS: Secrets OPerationS.
+> Simple and flexible tool for managing secrets.
+> More information: <https://github.com/mozilla/sops>.
+
+- Encrypt a file:
+
+`sops -e {{path/to/myfile.json}} > {{path/to/myfile.enc.json}}`
+
+- To decrypt a file in a `cat` fashion, use the `-d` flag:
```
Comment (zdroid) [inline pages/common/sops.md]: Do not use infinitive here

Comment (sbrl) [reply pages/common/sops.md]: might be worth including some extra information here, @zdroid. For example, we could include a suggested change?

File: pages/common/sops.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Change the extension of the file once encrypted:
+
+`sops -d --input-type json {{path/to/myfile.enc.json}}`
+
+- Extract keys by naming them, and array elements by numbering them:
+
+`sops -d --extract '["an_array"][1]' {{path/to/myfile.enc.json}}`
+
+- Diff two sops files:
```
Comment (zdroid) [inline pages/common/sops.md]:
```suggestion
Show the difference between two sops files
```

File: pages/common/sops.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+> SOPS: Secrets OPerationS.
+> Simple and flexible tool for managing secrets.
+> More information: <https://github.com/mozilla/sops>.
+
+- Encrypt a file:
+
+`sops -e {{path/to/myfile.json}} > {{path/to/myfile.enc.json}}`
+
+- Decrypt a file in `cat` fashion using the `-d` flag:
```
Comment (zdroid) [inline pages/common/sops.md]:
```suggestion
Decrypt a file to the standard output:
```
credits: @sbrl

File: pages/common/sops.md (hunk 2 above)
Comment (sbrl) [reply pages/common/sops.md]: You missed the `-` and the`:`, so I added then for you :-)

File: pages/common/sops.md (hunk 3 above)
Comment (sbrl) [reply pages/common/sops.md]: You missed the `-`, so I added it for you :-)

File: pages/common/sops.md (hunk 1 above)
Comment (mkatychev) [reply pages/common/sops.md]: @sbrl  Can you elaborate on what is the extra information?

**Pull request merged**